specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/examples/DetachADataDiskUsingToBeDetachedFlag.json

```diff
@@ -0,0 +1,65 @@
+{
+    "parameters": {
+      "subscriptionId": "{subscription-id}",
+      "resourceGroupName": "myResourceGroup",
+      "vmName": "myVM",
+      "api-version": "2019-03-01",
+      "parameters": {
+        "location": "westus",
+        "properties": {
+          "hardwareProfile": {
+            "vmSize": "Standard_D2_v2"
+          },
+          "storageProfile": {
+            "imageReference": {
+              "sku": "2016-Datacenter",
+              "publisher": "MicrosoftWindowsServer",
+              "version": "latest",
+              "offer": "WindowsServer"
+            },
+            "osDisk": {
+              "caching": "ReadWrite",
+              "managedDisk": {
+                "storageAccountType": "Standard_LRS"
+              },
+              "name": "myVMosdisk",
+              "createOption": "FromImage"
+            },
+            "dataDisks": [
+              {
+                "diskSizeGB": 1023,
+                "createOption": "Empty",
+                "lun": 0,
+                "toBeDetached": true
+              },
+              {
+                "diskSizeGB": 1023,
+                "createOption": "Empty",
```

Comment:
Pls. also specify the toBeDetached flag for the second data disk

Reply:
added